src/store.js

```diff
@@ -100,7 +100,7 @@ export class Store {
         handler(payload)
       })
     })
-    this._subscribers.forEach(sub => sub(mutation, this.state))
+    this._subscribers.slice().forEach(sub => sub(mutation, this.state))
```

Comment:
I'm open to cleaner/faster ways to generate a shallow copy.

Reply:
I guess it's ok to use `slice` for that.